build-tools/packages/build-cli/src/flags.ts

```diff
@@ -191,3 +210,70 @@ export const skipCheckFlag = Flags.boolean({
 	description: "Skip all checks.",
 	exclusive: ["install", "commit", "branchCheck", "updateCheck", "policyCheck"],
 });
+
+/**
+ * A set of flags that can be used to select packages in the repo. These flags provide a common way for commands to
+ * implement package selection and filtering.
+ */
+export const selectionFlags = {
+	all: Flags.boolean({
+		description:
+			"Run on all packages and release groups. Cannot be used with --dir, --packages, or --releaseGroup.",
+		exclusive: ["dir", "packages", "releaseGroup"],
+		helpGroup: "PACKAGE SELECTION",
+	}),
+	dir: Flags.directory({
+		description:
+			"Run on the package in this directory. Cannot be used with --all, --packages, or --releaseGroup.",
+		exclusive: ["packages", "releaseGroup", "all"],
+		helpGroup: "PACKAGE SELECTION",
+	}),
+	packages: Flags.boolean({
+		description:
+			"Run on all independent packages in the repo. Cannot be used with --all, --dir, or --releaseGroup.",
+		default: false,
+		exclusive: ["dir", "releaseGroup", "all"],
+		helpGroup: "PACKAGE SELECTION",
+	}),
+	releaseGroup: releaseGroupWithAllFlag({
+		description:
+			"Run on all packages within the release group. Cannot be used with --all, --dir, or --packages. This does not include release group root packages; to include those as well, use the --releaseGroupRoots argument.",
+		exclusive: ["all", "dir", "packages"],
+		helpGroup: "PACKAGE SELECTION",
+		multiple: true,
+	}),
+	releaseGroupRoot: releaseGroupWithAllFlag({
+		description:
+			"Run on the root package of the specified release groups. Cannot be used with --all, --dir, or --packages.",
```

Comment:
A bit unclear if `releaseGroupRoot` runs things _only_ for the root package of the release group, or in the root _and_ the other packages as suggested by the docs in `releaseGroup` ("to include those as well").

Reply:
I updated the text to hopefully clarify this further. They're separate flags, so if you really do want to run something on all the packages in a release group you need to use both `--releaseGroup` and `--releaseGroupRoot`.